lib/src/cache/cache.dart

```diff
@@ -0,0 +1,237 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import 'package:w_common/disposable.dart';
+import 'package:w_common/func.dart';
+
+/// Immutable payload that indicates a change in a [Cache].
+class CacheContext<TIdentifier, TValue> {
+  /// Identifies a single instance of a [TValue]
```

Comment:
For future reference dart doc will format anything surrounded by brackets that it can't link as inline code. 

<img width="401" alt="screen shot 2017-01-04 at 4 14 31 pm" src="https://cloud.githubusercontent.com/assets/15633954/21662783/1eeccc4c-d299-11e6-8b6c-87533c735c9f.png">

looks good to me 👍